packages/terra-data-grid/src/index.js

```diff
@@ -1,4 +1,5 @@
 import WorklistDataGrid from './WorklistDataGrid';
+import DataGrid from './DataGrid';
 
 // eslint-disable-next-line import/prefer-default-export
-export { WorklistDataGrid };
```

Comment:
I don't think we're exporting DataGrid for consumers at this time.

Reply:
Correct, we are not exposing the DataGrid component to consumers.